src/SFA.DAS.ReferenceData.Api/Controllers/OrganisationController.cs

```diff
@@ -87,8 +90,15 @@ public OrganisationController(IMediator mediator)
                 SearchTerm = searchTerm,
                 MaximumResults = maximumResults
             };
-
-            var response = await _mediator.SendAsync(query);
+            SearchOrganisationsResponse response;
+            try
+            {
+                response = await _mediator.SendAsync(query);
+            }
+            catch (Exception e)
```

Comment:
never been a fan of catching a generic exception, too heavy handed for me. Can you not check for more specific exceptions that you know will need to be handled and then let the rest bubble up the stack?

Reply:
This being a true external service we have no control over what errors can or do occur. the application behaviour required is that a zero length list is returned to all client SFA applications so that the correct business flow can be obtained without alarming the end user. This was the logical place to trap and deal with it.